internal/cmd/base/servers.go

```diff
@@ -620,7 +620,7 @@ func (b *Server) CreateGlobalKmsKeys(ctx context.Context) error {
 
 func (b *Server) DestroyDevDatabase() error {
 	if b.Database != nil {
-		underlyingDB, err := b.Database.DB()
+		underlyingDB, err := b.Database.SqlDB()
 		if err != nil {
 			return err
 		}
```

Comment:
I think this `Close` logic should probably be moved into a method on `db.DB`, ie:

```go
func (d *DB) Close() error {
  underlyingDB, err := d.SqlDB()
  if err != nil {
    return err
  }
  return underlying.Close()
}
```
}

Reply:
ty!